### PR TITLE
Constructs imageable route concern in image serializer

### PR DESCRIPTION
**Pull request by nomadicoder**
Issue: #MAN-308

### CODE REVIEW

File: app/serializers/concerns/image_serializable.rb [hunk 1]
```diff
@@ -5,11 +5,11 @@ module ImageSerializable
 
   included do
     attribute :image, if: Proc.new { |the_object| the_object.image.attached? } do |the_object|
-      helpers.rails_blob_url(the_object.image)
+      eval "helpers.#{the_object.class.to_s.underscore}_image_large_url(the_object)"
```
Comment (bibliotechy) [inline app/serializers/concerns/image_serializable.rb]: I'm pretty sure `send` is preferable to eval for safety and performance reasons.
```suggestion
      helpers.send("#{the_object.class.to_s.underscore}_image_large_url", the_object)
```

File: app/serializers/concerns/image_serializable.rb [hunk 2]
```diff
@@ -5,11 +5,11 @@ module ImageSerializable
...
     end
 
     attribute :thumbnail_image, if: Proc.new { |the_object| the_object.image.attached? } do |the_object|
-      helpers.rails_representation_url(the_object.index_image)
+      eval "helpers.#{the_object.class.to_s.underscore}_image_thumbnail_url(the_object)"
```
Comment (bibliotechy) [inline app/serializers/concerns/image_serializable.rb]:
```suggestion
      helpers.send("#{the_object.class.to_s.underscore}_image_thumbnail_url", the_object)
```

File: app/serializers/concerns/image_serializable.rb [hunk 3]
```diff
@@ -5,11 +5,11 @@ module ImageSerializable
 
   included do
     attribute :image, if: Proc.new { |the_object| the_object.image.attached? } do |the_object|
-      helpers.rails_blob_url(the_object.image)
+      helpers.send("#{the_object.class.to_s.underscore}_image_large_url", the_object)
     end
 
     attribute :thumbnail_image, if: Proc.new { |the_object| the_object.image.attached? } do |the_object|
-      helpers.rails_representation_url(the_object.index_image)
+      helpers.send("#{the_object.class.to_s.underscore}_image_large_url", the_object)
```
Comment (bibliotechy) [inline app/serializers/concerns/image_serializable.rb]: this should be `_image_thumbnail_url`I think?

Comment (nomadicoder) [reply app/serializers/concerns/image_serializable.rb]: Fixed

**Pull request merged**